packages/cli/src/requests.ts

```diff
@@ -587,4 +587,6 @@ export declare namespace AiAssistantRequest {
 
 	type SuggestionPayload = { sessionId: string; suggestionId: string };
 	type ApplySuggestion = AuthenticatedRequest<{}, {}, SuggestionPayload>;
+	//@ts-ignore
```

Comment:
why? Could we fix this?

Reply:
that was only temporary while waiting for the SDK to be released